civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,198 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom:0px;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0px;
+    position: absolute;
+    top: 0px;
+  }
+
+  .no-user-image {
+    background: url(/sites/all/modules/civihr-custom/civihr_employee_portal/images/profile-default.png) center no-repeat;
+    background-size: 80px 80px;
+    overflow: hidden;
+    text-indent: -1000px;
+  }
+
+  .views-field-image-URL img,
+  .no-user-image {
+    border-radius: 50%;
+    height: 80px;
+    width: 80px;
+  }
+
+  .chr_panel__footer {
+    border-top: none;
+    margin-top: 0px;
+  }
+
+  .views-field-nothing .field-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    white-space: nowrap;
+    width: 100%;
+    > * {
+      margin-right: 1rem;
+    }
+  }
+
+  .views-field-ni-ssn-70 .field-content {
+    width: 100%;
+  }
+
+  .nsi-exists.nsi-application-in-progress-1 {
+    > * {
+      display: none;
+    }
+
+    .nsi-application-status {
+      display: block;
+    }
+  }
+
+  div.views-field-display-name {
+    color: $crm-gray-darker-alt;
+    font-size: $crm-font-size-large;
+    margin-bottom: 5px;
+
+    .views-label {
+      display: none;
+    }
+
+    > * {
+      width: 100%;
+    }
+  }
+
+  .nsi-application-status > * {
+    display: inline-block;
+    float: left;
+    width: 50%;
+  }
+
+  .nsi-application-status.not-applying,
+  .nsi-application-status.No {
+    display: none;
+  }
+
+  .chr_actions-wrapper a {
+    margin: 1rem .5rem;
+  }
+}
+
+.view-mydetails-personal,
+.view-mydetails-myaddress,
+.view-id-mydetails_myrole,
+.view-mydetails-payrollinformation {
+  @extend .my-details-view-panel;
```

Comment:
*never* use `@extend` unless you have a very good reason and it's fully explained and justified

Reply:
Fine, but this is a group of styles that I want to replicate, and I don't want to alter anything else. So, as far as I know it can be done with a mixin, extend or repeating code. What do you suggest me to do?